lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -1004,6 +1060,7 @@ def main():
         template=dict(default=None),
         template_version=dict(default=None, type='int'),
         use_latest_template_version=dict(default=None, type='bool'),
+        storage_domain=dict(default=None),
         disks=dict(default=[], type='list'),
```

Comment:
Missing

`disk_format=dict(default='COW', choices='COW, 'RAW'),`

This makes me think that changing the disk_format may not have been tested...